server/conf/application.conf

```diff
@@ -517,3 +517,9 @@ api_secret_salt = ${?CIVIFORM_API_SECRET_SALT}
 
 api_keys_ban_global_subnet = true
 api_keys_ban_global_subnet = ${?CIVIFORM_API_KEYS_BAN_GLOBAL_SUBNET}
+
+# Max and default page size for ProgramApplicationsApiController.list
+# This should be kept low enough so that the request completes before hitting the
+# load balancer timeout.
+api_applications_list_max_page_size = 1000
```

Comment:
> do we think different deployments will really want different system wide defaults, much more so than individuals wanting customization as they interact with a specific program?

I don't know. I made this value configurable so that if a deployment is having a hard time with an API consumer requesting pages that are too big, they can reduce the max size without code changes (and doing this much was easy). If more granular control is needed I think we should let feedback drive the requirements for it.

Reply:
I'm ok leaving it, but it does feel like the wrong place for the configuration.  The folks using the Admin interface are unlikely to be aware of it.